src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToTargetEolFrameworks.cs

```diff
@@ -75,7 +75,7 @@ public void It_does_not_warn_when_deactivating_check()
             var testProject = new TestProject()
             {
                 Name = $"EolNoWarning",
-                TargetFrameworks = "netcoreapp1.0",
+                TargetFrameworks = ToolsetInfo.CurrentTargetFramework,
```

Comment:
This test is specifically targeting an out of support TFM so should not be changed.